pandas/core/groupby/generic.py

```diff
@@ -791,11 +824,8 @@ def _aggregate_multiple_funcs(self, arg, _level):
             # list of functions / function names
             columns = []
             for f in arg:
-                if isinstance(f, str):
-                    columns.append(f)
-                else:
-                    # protect against callables without names
-                    columns.append(com.get_callable_name(f))
+                columns.append(com.get_callable_name(f) or f)
```

Comment:
Did we have to get rid of the `if...else` to make this work?

Reply:
this is ok as it is what get_callable_name does